tools/src/test/python/dlpx/virtualization/_internal/fake_plugin/direct/multiple_warnings.py

```diff
@@ -40,12 +42,12 @@ def mount_specification(repository, virtual_source):
 
 
 @vfiles.virtual.post_snapshot()
-def postSnapshot(repository, source_config, virtual_source):
+def post_snapshot(repository, source_config, virtual_source):
     return None
 
 
 @vfiles.virtual.pre_snapshot()
-def preSnapshot(repository, source_config, virtual_source):
+def pre_snapshot(repository, source_config, virtual_source):
```

Comment:
Should we add `snapshot_parameters` to these pre/post snapshot functions too?

Reply:
again this is virtual not linked.